packages/networks/types.ts

```diff
@@ -25,6 +25,7 @@ interface NetworkInfoBase {
   contractExplorer: string;
   testnet: boolean;
   backendEndpoint: string;
+  launchpadEndpoint: string;
```

Comment:
can we use a feature object instead?

- the base network object is already too heavy
- this allows to check for presence of all required params at once